demo/helm/rook-operator/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for rook-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+image:
+  repository: quay.io/rook/rookd
+  tag: master-latest
+  pullPolicy: IfNotPresent
+
+resources:
+  limits:
+    cpu: 100m
```

Comment:
Yes, that is the chart generator default. Does it seem in the right ballpark for the operator?